src/Tribe/Template.php

```diff
@@ -402,27 +402,27 @@ protected function get_template_public_path( $base ) {
 	 * @return array
 	 */
 	protected function get_template_path_list() {
-		$folders = array();
+		$folders = [];
 
 		// Only look into public folders if we tell to use folders
 		if ( $this->template_folder_lookup ) {
-			$folders[] = array(
+			$folders[] = [
 				'id' => 'child-theme',
 				'priority' => 10,
 				'path' => $this->get_template_public_path( STYLESHEETPATH ),
-			);
-			$folders[] = array(
+			];
+			$folders[] = [
 				'id' => 'parent-theme',
 				'priority' => 15,
 				'path' => $this->get_template_public_path( TEMPLATEPATH ),
-			);
+			];
 		}
 
-		$folders[] = array(
+		$folders[] = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 8b06b2ca88df2e66561ce57ad7b475db</sub>